wordpress-mobile/WordPress-iOS
# Zendesk Mobile: show notification alert on Me > Help & Support

Fixes #9328 

When a push notification is received from Zendesk (i.e. when an agent replies to a ticket):
- On the Me tab, the 'Help & Support' row will display an indicator.
- On the 'Help & Support' view, the 'My Tickets' row will display an indicator.
- Both indicators are removed after 'My Tickets' is viewed.

**To test:**
- Go to Me > Help & Support > 'Contact Us' > submit a ticket.
  - Warning: You cannot use the email address associated with your Zendesk agent account (if you have one).

---

- Stay on the Support view.
- Leave the app open.
- Go to Zendesk and reply to your ticket.
- When the PN is received: (You'll see `Received push notification` in the log)
  - Verify the indicator appears on the 'My Tickets' row. (sometimes this takes a few seconds)
  - Go back to the Me tab.
    - Verify the indicator appears on the 'Help & Support' row.
  - Select 'Help & Support' to return to the Support view.
  - Select 'My Tickets', then close it.
    - Verify the indicator is removed from 'My Tickets' and back on 'Help & Support'.

---

Verify the indicators appear and are cleared for the following scenarios:
- On Support view, app minimized.
- On Me tab, app open.
- On Me tab, app minimized.
- On any other tab, app open. (the Me icon will change to the star when PN received)
- On any other tab, app minimized. (the Me icon will change to the star when PN received)

---

**My Tickets indicator:**
<img width="716" alt="my_tix" src="https://user-images.githubusercontent.com/1816888/39947099-e4945218-552d-11e8-86b2-0cc325ef5835.png">

**Help & Support indicator:**
<img width="718" alt="h_s" src="https://user-images.githubusercontent.com/1816888/39947083-daaf1f26-552d-11e8-8dec-a7c90bb33662.png">

---

Sorry @aerych - I was going to do these indicators separately, but then I realized they should be dependent on each other. Thanks for your time!
